crates/spawner/src/spawner.rs

```diff
@@ -83,6 +85,13 @@ fn spawn(
                 match active_type {
                     ActiveObjectType::Building(_) => {
                         entity_commands.insert(StaticSolid);
+
+                        let half_size: Vec2 =
+                            solid.ichnography().local_aabb().half_extents().into();
+                        entity_commands.insert(RectangleMarker::new(
```

Comment:
Wouldn't the following be a bit nicer API?

```rust
let local_aabb = solid.ichnography().local_aabb();
RectangleMarker::from_aabb_transform(local_aabb, transform)
```

Among other things, it would remove the necessity to define/import `RECTANGLE_MARKER_MARGIN` here.